app/payment-link-v2/reference-confirm/reference-confirm.controller.js

```diff
@@ -22,7 +23,10 @@ function getPage (req, res, next) {
 
   const sessionReferenceNumber = getSessionVariable(req, 'referenceNumber')
 
-  data.backLinkHref = replaceParamsInPath(paymentLinksV2.reference, product.externalId)
+  data.referencePageUrl = replaceParamsInPath(paymentLinksV2.reference, product.externalId)
+  data.confirmPageUrl = replaceParamsInPath(paymentLinksV2.confirm, product.externalId)
+
+  data.isThisAValidReferenceText = res.locals.__p('paymentLinksV2.referenceConfirm.areYouSureThisIsAValidReference').replace('%s', product.reference_label)
```

Comment:
This wasn't explored in the design for the new journey, so the design for this should be the same as the existing page for confirming the reference. On the existing page, the text for this is just "Are you sure this is a reference number?".

It might make sense to template this in, but should we stick to the existing content until it is reviewed?

Reply:
Sounds good - I will keep it the same for now.